### ISSUE TITLE
[ANDROID] The AnalyticsExample Project don't works.

### ISSUE THREAD

**Issue by gtoto007** [fuse-compound/Fuse.Firebase#81]
Fuse 1.8.1 Mac

I tried to launch the project `AnalyticsExample` for Android  with this command:
`fuse build -t=android -r`
 and I have this error:
```
....
/Users/r0xstation/Fuse.Firebase/AnalyticsExample/build/Android/Debug/app/.externalNativeBuild/cmake/debug/obj/armeabi-v7a/libAnalyticsExample.so
:app:compileDebugSources
:app:mergeDebugShaders
:app:compileDebugShaders
:app:generateDebugAssets
:app:mergeDebugAssets
:app:transformClassesWithDexForDebug FAILED
```

For IOS it's works now thank you.

All log:
```
MacBook-Pro-di-admin:AnalyticsExample r0xstation$ fuse build -t=android -r
Uno 1.8.0 (build 6680) macOS 10.13 x86_64 6c6e84c
Copyright © 2015-2018 Fusetools


Configuring
  0.52 s

Compiling syntax tree
  1.80 s
Generating code and data
Fontconfig warning: no <cachedir> elements found. Check configuration.
Fontconfig warning: adding <cachedir>~/Library/Caches/com.xamarin.fontconfig</cachedir>
Fontconfig warning: adding <cachedir prefix="xdg">fontconfig</cachedir>

  4.68 s
Building Android app

Download https://jcenter.bintray.com/com/google/gms/google-services/3.0.0/google-services-3.0.0.pom

Download https://jcenter.bintray.com/com/google/gms/google-services/3.0.0/google-services-3.0.0.jar

Download https://maven.google.com/com/google/firebase/firebase-core/11.8.0/firebase-core-11.8.0.pom

Download https://maven.google.com/com/google/firebase/firebase-analytics/11.8.0/firebase-analytics-11.8.0.pom

Download https://maven.google.com/com/google/firebase/firebase-analytics-impl/11.8.0/firebase-analytics-impl-11.8.0.pom

Download https://maven.google.com/com/google/firebase/firebase-analytics-license/11.8.0/firebase-analytics-license-11.8.0.pom

Download https://maven.google.com/com/google/firebase/firebase-iid/11.8.0/firebase-iid-11.8.0.pom

Download https://maven.google.com/com/google/firebase/firebase-analytics-impl-license/11.8.0/firebase-analytics-impl-license-11.8.0.pom

Download https://maven.google.com/com/google/firebase/firebase-iid-license/11.8.0/firebase-iid-license-11.8.0.pom

Download https://maven.google.com/com/google/android/gms/play-services-basement/11.8.0/play-services-basement-11.8.0.pom

Download https://maven.google.com/com/google/android/gms/play-services-basement-license/11.8.0/play-services-basement-license-11.8.0.pom

Download https://maven.google.com/com/google/firebase/firebase-common/11.8.0/firebase-common-11.8.0.pom

Download https://maven.google.com/com/google/firebase/firebase-common-license/11.8.0/firebase-common-license-11.8.0.pom

Download https://maven.google.com/com/google/android/gms/play-services-tasks/11.8.0/play-services-tasks-11.8.0.pom

Download https://maven.google.com/com/google/android/gms/play-services-tasks-license/11.8.0/play-services-tasks-license-11.8.0.pom

Download https://maven.google.com/com/google/android/gms/play-services-basement-license/11.8.0/play-services-basement-license-11.8.0.aar

Download https://maven.google.com/com/google/android/gms/play-services-basement/11.8.0/play-services-basement-11.8.0.aar

Download https://maven.google.com/com/google/android/gms/play-services-tasks-license/11.8.0/play-services-tasks-license-11.8.0.aar

Download https://maven.google.com/com/google/android/gms/play-services-tasks/11.8.0/play-services-tasks-11.8.0.aar

Download https://maven.google.com/com/google/firebase/firebase-common-license/11.8.0/firebase-common-license-11.8.0.aar

Download https://maven.google.com/com/google/firebase/firebase-common/11.8.0/firebase-common-11.8.0.aar

Download https://maven.google.com/com/google/firebase/firebase-iid-license/11.8.0/firebase-iid-license-11.8.0.aar

Download https://maven.google.com/com/google/firebase/firebase-iid/11.8.0/firebase-iid-11.8.0.aar

Download https://maven.google.com/com/google/firebase/firebase-analytics-impl-license/11.8.0/firebase-analytics-impl-license-11.8.0.aar

Download https://maven.google.com/com/google/firebase/firebase-analytics-impl/11.8.0/firebase-analytics-impl-11.8.0.aar

Download https://maven.google.com/com/google/firebase/firebase-analytics-license/11.8.0/firebase-analytics-license-11.8.0.aar

Download https://maven.google.com/com/google/firebase/firebase-analytics/11.8.0/firebase-analytics-11.8.0.aar

Download https://maven.google.com/com/google/firebase/firebase-core/11.8.0/firebase-core-11.8.0.aar

Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformClassesWithDexForDebug'.
> com.android.build.api.transform.TransformException: com.android.ide.common.process.ProcessException: java.util.concurrent.ExecutionException: com.android.dex.DexException: Multiple dex files define Lcom/google/android/gms/common/internal/zzag;

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

(unknown): E0200: Android build failed

Download https://jcenter.bintray.com/com/google/gms/google-services/3.0.0/google-services-3.0.0.pom
Download https://jcenter.bintray.com/com/google/gms/google-services/3.0.0/google-services-3.0.0.jar
Download https://maven.google.com/com/google/firebase/firebase-core/11.8.0/firebase-core-11.8.0.pom
Download https://maven.google.com/com/google/firebase/firebase-analytics/11.8.0/firebase-analytics-11.8.0.pom
Download https://maven.google.com/com/google/firebase/firebase-analytics-impl/11.8.0/firebase-analytics-impl-11.8.0.pom
Download https://maven.google.com/com/google/firebase/firebase-analytics-license/11.8.0/firebase-analytics-license-11.8.0.pom
Download https://maven.google.com/com/google/firebase/firebase-iid/11.8.0/firebase-iid-11.8.0.pom
Download https://maven.google.com/com/google/firebase/firebase-analytics-impl-license/11.8.0/firebase-analytics-impl-license-11.8.0.pom
Download https://maven.google.com/com/google/firebase/firebase-iid-license/11.8.0/firebase-iid-license-11.8.0.pom
Download https://maven.google.com/com/google/android/gms/play-services-basement/11.8.0/play-services-basement-11.8.0.pom
Download https://maven.google.com/com/google/android/gms/play-services-basement-license/11.8.0/play-services-basement-license-11.8.0.pom
Download https://maven.google.com/com/google/firebase/firebase-common/11.8.0/firebase-common-11.8.0.pom
Download https://maven.google.com/com/google/firebase/firebase-common-license/11.8.0/firebase-common-license-11.8.0.pom
Download https://maven.google.com/com/google/android/gms/play-services-tasks/11.8.0/play-services-tasks-11.8.0.pom
Download https://maven.google.com/com/google/android/gms/play-services-tasks-license/11.8.0/play-services-tasks-license-11.8.0.pom
Download https://maven.google.com/com/google/android/gms/play-services-basement-license/11.8.0/play-services-basement-license-11.8.0.aar
Download https://maven.google.com/com/google/android/gms/play-services-basement/11.8.0/play-services-basement-11.8.0.aar
Download https://maven.google.com/com/google/android/gms/play-services-tasks-license/11.8.0/play-services-tasks-license-11.8.0.aar
Download https://maven.google.com/com/google/android/gms/play-services-tasks/11.8.0/play-services-tasks-11.8.0.aar
Download https://maven.google.com/com/google/firebase/firebase-common-license/11.8.0/firebase-common-license-11.8.0.aar
Download https://maven.google.com/com/google/firebase/firebase-common/11.8.0/firebase-common-11.8.0.aar
Download https://maven.google.com/com/google/firebase/firebase-iid-license/11.8.0/firebase-iid-license-11.8.0.aar
Download https://maven.google.com/com/google/firebase/firebase-iid/11.8.0/firebase-iid-11.8.0.aar
Download https://maven.google.com/com/google/firebase/firebase-analytics-impl-license/11.8.0/firebase-analytics-impl-license-11.8.0.aar
Download https://maven.google.com/com/google/firebase/firebase-analytics-impl/11.8.0/firebase-analytics-impl-11.8.0.aar
Download https://maven.google.com/com/google/firebase/firebase-analytics-license/11.8.0/firebase-analytics-license-11.8.0.aar
Download https://maven.google.com/com/google/firebase/firebase-analytics/11.8.0/firebase-analytics-11.8.0.aar
Download https://maven.google.com/com/google/firebase/firebase-core/11.8.0/firebase-core-11.8.0.aar
Incremental java compilation is an incubating feature.
:app:preBuild UP-TO-DATE
:app:preDebugBuild UP-TO-DATE
:app:checkDebugManifest
:app:preReleaseBuild UP-TO-DATE
:app:prepareComAndroidSupportAnimatedVectorDrawable2340Library
:app:prepareComAndroidSupportAppcompatV72340Library
:app:prepareComAndroidSupportDesign2340Library
:app:prepareComAndroidSupportRecyclerviewV72340Library
:app:prepareComAndroidSupportSupportCompat2520Library
:app:prepareComAndroidSupportSupportCoreUi2520Library
:app:prepareComAndroidSupportSupportCoreUtils2520Library
:app:prepareComAndroidSupportSupportFragment2520Library
:app:prepareComAndroidSupportSupportMediaCompat2520Library
:app:prepareComAndroidSupportSupportV42520Library
:app:prepareComAndroidSupportSupportVectorDrawable2340Library
:app:prepareComGoogleAndroidGmsPlayServicesBase920Library
:app:prepareComGoogleAndroidGmsPlayServicesBasement1180Library
:app:prepareComGoogleAndroidGmsPlayServicesBasementLicense1180Library
:app:prepareComGoogleAndroidGmsPlayServicesTasks1180Library
:app:prepareComGoogleAndroidGmsPlayServicesTasksLicense1180Library
:app:prepareComGoogleFirebaseFirebaseAnalytics1180Library
:app:prepareComGoogleFirebaseFirebaseAnalyticsImpl1180Library
:app:prepareComGoogleFirebaseFirebaseAnalyticsImplLicense1180Library
:app:prepareComGoogleFirebaseFirebaseAnalyticsLicense1180Library
:app:prepareComGoogleFirebaseFirebaseCommon1180Library
:app:prepareComGoogleFirebaseFirebaseCommonLicense1180Library
:app:prepareComGoogleFirebaseFirebaseCore1180Library
:app:prepareComGoogleFirebaseFirebaseDatabase920Library
:app:prepareComGoogleFirebaseFirebaseDatabaseConnection920Library
:app:prepareComGoogleFirebaseFirebaseIid1180Library
:app:prepareComGoogleFirebaseFirebaseIidLicense1180Library
:app:prepareDebugDependencies
:app:compileDebugAidl
:app:compileDebugRenderscript
:app:generateDebugBuildConfig
:app:generateDebugResValues
:app:processDebugGoogleServices
Parsing json file: /Users/r0xstation/Fuse.Firebase/AnalyticsExample/build/Android/Debug/app/google-services.json
:app:generateDebugResources
:app:mergeDebugResources
:app:processDebugManifest
:app:processDebugResources
:app:generateDebugSources
:app:incrementalDebugJavaCompilationSafeguard
:app:compileDebugJavaWithJavac
:app:compileDebugJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
:app:generateJsonModelDebug
:app:externalNativeBuildDebug
  building /Users/r0xstation/Fuse.Firebase/AnalyticsExample/build/Android/Debug/app/.externalNativeBuild/cmake/debug/obj/armeabi-v7a/libAnalyticsExample.so
:app:compileDebugSources
:app:mergeDebugShaders
:app:compileDebugShaders
:app:generateDebugAssets
:app:mergeDebugAssets
:app:transformClassesWithDexForDebug FAILED

BUILD FAILED

```

**Comment by LuisRodriguezLD** [comment]
Probably related the same Firebase SDK update. I do not have an Android but I'll try to make it build at least...

**Comment by gtoto007** [comment]
If you want a help to test the android build, I can help you.

**Comment by LuisRodriguezLD** [comment]
Sounds great, I'll give it a go later today!

**Comment by LuisRodriguezLD** [comment]
So I found the problem, the` Database` module was using the same version as the `Core` but a different version than the `Analytics`. [This branch](https://github.com/fuse-compound/Fuse.Firebase/tree/hotfix/broken-analytics) has all modules updated to 11.8.0 and it fixes this issue. 

However, I am a little afraid as this changes might break something. Can you help me test the examples and make sure they _still_ work so I can merge this update?

**Comment by ckarmy** [comment]
With the [branch](https://github.com/fuse-compound/Fuse.Firebase/tree/hotfix/broken-analytics) I'm still getting the same error...

**Comment by ckarmy** [comment]
[With the branch](https://github.com/fuse-compound/Fuse.Firebase/tree/hotfix/broken-analytics) work but my app crash when FBAnalytics.logEvent(FirebaseAnalytics.Event.SELECT_CONTENT, p);

**Comment by LuisRodriguezLD** [comment]
Also fixed with #86
